server/test/mocks/movementsInReceptionMock.ts

```diff
@@ -0,0 +1,19 @@
+import { OffenderInReception } from '../../data/interfaces/offenderInReception'
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Is this not working? https://github.com/ministryofjustice/hmpps-digital-prison-services/blob/main/.eslintrc.json#L106